### DISCUSSION TITLE
Methods iterator in sootup

### DISCUSSION THREAD

**Discussion by vskrishnan27** [soot-oss/SootUp#696] [General]
SootMethod method;
for (SootClass c : Scene.v().getApplicationClasses()) {
    if(c.getName().equals("example.HelloWorld")){
        for (SootMethod m : c.getMethods()) {
            if (!m.hasActiveBody()) {
                continue;
            }
            if (m.getName().equals("entryPoint")) {
                method = m;
                break;
            }
        }
    }
}

**Comment by swissiety** [comment]
have a look at the getting started https://soot-oss.github.io/SootUp/getting-started/#retrieving-a-method

**Comment by vskrishnan27** [comment]
hi swissiety,
i have used 1.0.0 jars so im unable to find method iterator now im using latest sootup jars now i can iterate methods from a class
